src/MoBi.Presentation/Tasks/Interaction/InteractionTaskFor.cs

```diff
@@ -253,7 +253,7 @@ private void throwAddedEvent(TChild newObjectBase, IObjectBase parent)
          Context.PublishEvent(new AddedEvent<TChild>(newObjectBase, parent));
       }
 
-      private bool correctName(TChild child, TParent parent)
+      protected virtual bool CorrectName(TChild child, TParent parent)
```

Comment:
I need to override this default renaming in the case of Expression. They have a specific naming requirement